samples/js-customer-registry/customer_view.proto

```diff
@@ -140,7 +140,7 @@ service CustomerByNameView {
 
   rpc IgnoreOtherEvents(google.protobuf.Any) returns (domain.CustomerState) {
     option (akkaserverless.method).eventing.in = {
-      event_sourced_entity: "customers"
+      event_sourced_entity: "customers" // <7>
```

Comment:
Might be better with the callout on the rpc line instead, like it is for <1> and <4>

```suggestion
      event_sourced_entity: "customers"
```